### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_DeployDacpac_DeployDacpacSummaryScreen_ScreenReader: Nvda  does not announce the 2nd row value.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6688]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS; #A11yTCS; #A11y_ADS_Schema_Dacpac_Backup; #Azure Data Studio; #DesktopApp; #Win10; #NVDA; #MAS1.3.1; #MAS4.2.3;

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.9.0 (user setup)
Commit: 78a42e1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
Nvda Version: 2019.2rc2

**Additional Details:**
Tools Name: Nvda
MAS References: MAS1.3.1, MAS4.2.3

**Repro Steps:**
1.Launch Azure Data Studio application.
2.Connect to server with windows credentials.
3.Navigate to Extensions from left menu.
4.Search & install Dacpac extension.
5.Navigate to Connections from left menu.
6.Expand the server connected in step 2.
7.Right click an on prem database and select Data-tier application wizard from Context menu.
8.Select first radio button.
9.Navigate till next button & hit enter.
10.Provide all the details for Deploy Dacpac & hit next.
11.Navigate till Next button & hit enter.
12.Start screen reader as Nvda.
13.Navigate to the second row using arrow key arrow.

**Actual:**
**Issue1:**
On navigating to the second row below the header row using down key arrow nvda does not announce the row value. It announces "Setting column 1 Target Server"

**Issue2:**
On navigating to the header row it announces it as row 1, when u move to the third row of the table it announces it as row 2 and the fourth row is announced as row 3.

**Same issue is observed in the following pages:**
1.Extract Dacpac Summary Screen
Follow the exact same steps mentioned above but select the second radio button.
2.Import Bacpac Summary Screen
Follow the exact same steps mentioned above but select the third radio button.
3. Export Bacpac Summary Screen
Follow the exact same steps mentioned above but select the 4th radio button and click on next button

**Expected:**
**Issue1:**
On navigating to the second row below the header row using down key arrow nvda should announce the row value as 2. The announcement should be "Row 2 Setting column 1 Target Server"

**Issue2:**
On navigating to the header row it announces it as row 1,since the table has 4 rows when navigated to the next rows under header row in the table it should have the next ordered number row 2 followed by row 3 and row 4. 

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: (https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Screen reader users will not know the exact numbers of rows present in the table as well as even though the data of row 2 is announced they wont know it belongs to row 2 . 

**MAS Reference:**
MAS 1.3.1- https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4} 
MAS4.2.3-
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={30e4fb40-dcc1-45b4-b714-5e2c28b23ae8}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={30e4fb40-dcc1-45b4-b714-5e2c28b23ae8})

**Attachment for Reference:**
[[Accessibility]A11y_AzureDataStudio(Schema,Dacpac,Backup)_DeployDacpac_DeployDacpacSummaryScreen_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3491130/Accessibility.A11y_AzureDataStudio.Schema.Dacpac.Backup._DeployDacpac_DeployDacpacSummaryScreen_ScreenReader.pptx)

**Comment by udeeshagautam** [comment]
Seems upstream for combination of SlickGrid,  given that role=presentation is not taken as a non-row value for NVDA and that is the indication slick grid provides for header row. https://github.com/nvaccess/nvda/issues/5717

**Comment by hannahqin** [comment]
Reopening and reassigning to @fsteffi as per the [Accessibility External Bug Process](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx). 

@fsteffi, this is behavior that stems from NVDA: see nvaccess/nvda#5717 for discussion of a related issue that was closed. Since nvaccess/nvda#5717 was closed with no changes, can we deem this to be expected NVDA behavior and close this issue?

**Comment by hannahqin** [comment]
There's a related issue that was closed and approved by an SME for exclusion here: http://sqlbuvsts01:8080/Main/SQL%20Server/_workitems#id=11688052&triage=true&fullScreen=false&_a=edit (Azure Data Studio was formerly called SQL Operations Studio). See discussion for details.

Can we close this one?

**Comment by fsteffi** [comment]
Closing this bug as discussed with the SME.
